### PR TITLE
Added button changes and select/de-select All

### PR DESCRIPTION
**Pull request by HarikaMunipalle**

### CODE REVIEW

File: assets/js/facets.js [hunk 1]
```diff
@@ -46,7 +46,28 @@ $.getJSON(window.federalist.path.baseurl + '/search.json', function(res) {
     $('label[for="career-competency-select-all"]').css( "outline", "none" );
   });
 
-  $("#job-career-competency-select-all, #general-career-competency-select-all").on('change', function() {
+    $("#job-career-competency-select-all, #general-career-competency-select-all").on('change', function () {
```
Comment (alexdntowe) [inline assets/js/facets.js:49]: Add comments

File: assets/js/facets.js [hunk 2]
```diff
@@ -46,7 +46,28 @@ $.getJSON(window.federalist.path.baseurl + '/search.json', function(res) {
...
+        var id = this.id;
+        if (id == 'job-career-competency-select-all') {
+            var jobSelect = '#job-career-competency-select';
+            if ($(jobSelect).text() == 'Select All') {
+                $(jobSelect).html("<strong>De-Select All</strong>");
+                $('[data-filter="competency"][data-major-group="job-specific"]').forEach(item => {
+/*                    let itemElement = createId(item);*/
+                    console.log(item);
```
Comment (alexdntowe) [inline assets/js/facets.js:57]: remove console log

File: assets/js/facets.js [hunk 3]
```diff
@@ -127,7 +148,14 @@ $.getJSON(window.federalist.path.baseurl + '/search.json', function(res) {
       });
 
       $("#" + eventId).on('change', function () {
-        console.log("Select All Competency Group");
+          console.log("Select All Competency Group");
```
Comment (alexdntowe) [inline assets/js/facets.js:151]: remove console log

File: career-planning-tool/index.html [hunk 4]
```diff
@@ -77,20 +77,38 @@ <h3>GS Level</h3>
...
                             {% assign level_id = level | downcase | replace: ", ", "-" | replace: " ", "-" %}
+                             {% if level contains '7-9' %}
                             <button
                                 type="button"
                                 id="GS-{{ level_id }}"
                                 data-id="GS-{{ level_id }}"
                                 title="GS Level - {{ level }}"
                                 class="usa-button padding-1 margin-bottom-2 margin-right-0 text-bold text-black border-2px bg-white radius-lg border-gray-30"
-                            >GS {{ level }}</button>
+                            >Entry (GS 7,8,9)</button>
```
Comment (alexdntowe) [inline career-planning-tool/index.html:87]: Not needed

Comment (alexdntowe) [reply career-planning-tool/index.html:87]: Already addressed

File: assets/js/facets.js (hunk 1 above)
Comment (HarikaMunipalle) [reply assets/js/facets.js:49]: spacing issue

**Pull request closed without merging**